Algorithms/Compressors/HuffmanCompressor.cs

```diff
@@ -67,7 +72,11 @@ public HuffmanCompressor(ISorter<ListNode> sorter, Translator translator)
                 AddMany(decompressionKeys, lsdk.Select(kvp => ("0" + kvp.Key, kvp.Value)));
             }
 
-            if (tree.RightChild != null)
```

Comment:
I'd suggest not inverting this condition to be consistent with the way LeftChild is processed.